### DISCUSSION TITLE
On td-agent force-restart getting log message posted as : error writing passwd entry: Broken pipe

### DISCUSSION THREAD

**Discussion by jmeher2020** [fluent/fluentd#4740] [Q&A]
### Describe the bug

On td-agent force-restart getting log message posted as : error writing passwd entry: Broken pipe with tail td-agent.log file

uat:/# ./td-agent force-restart; tail -20f /log/ td-agent/td-agent.log
error writing passwd entry: Broken pipe
Restarting td-agent:
[ ok ] td-agent.

### To Reproduce

Unable to re-produce with new setup.

### Expected behavior

The td-agent should restart without message "error writing passwd entry: Broken pipe"

### Your Environment

```markdown
- Fluentd version: 1.10.2
- Package version: 3.7.1
- Operating system: PRETTY_NAME="Debian GNU/Linux 10 (buster)"
- Kernel version: 5.15.168generic
```

### Your Configuration

```apache
<match *.*.*>
  @type copy

  <store>
      @type elasticsearch
      hosts localhost
      port 9200
```

### Your Error Log

```shell
uat:/# ./td-agent force-restart; tail -20f /log/ td-agent/td-agent.log
error writing passwd entry: Broken pipe
Restarting td-agent:
[ ok ] td-agent.
2024-12-09 05:58:03 -0500 [warn]: #0 pattern not matched: "type=PROCTITLE msg=audit(1733741883.229:3963020):
```

**Comment by daipom** [comment]
As a bug fix, it would not be easy if it could be reproduced in a more recent version.
(Please check [the contributing guidelines](https://github.com/fluent/fluentd/blob/d5df992d844af38233c4d3788de714d644ddb6c4/CONTRIBUTING.md).)

So I have transformed this issue to Q&A.
